orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -0,0 +1,41 @@
+// This file contains c apis for on device training
```

Comment:
license

Reply:
this file does not need one because it is not a standalone file